front-end/src/Header.js

```diff
@@ -10,6 +12,8 @@ const Header = () => {
       header={
         <div className="header-root">
           <Hamburger pageWrapId={'page-wrap'} outerContainerId={'outer-container'} />
+          <LoginButton/>
+          <SignupButton/>
```

Comment:
I'd suggest moving these to after `h1` since we want them to be right-aligned, according to the wireframe. I also suggest grouping them in a `div`